libmproxy/console/common.py

```diff
@@ -161,6 +165,88 @@ def raw_format_flow(f, focus, extended, padding):
     pile.append(urwid.Columns(resp, dividechars=1))
     return urwid.Pile(pile)
 
+## common save body parts
+def save_body(path, master, state, content):
+    if not path:
+        return
+    state.last_saveload = path
+    path = os.path.expanduser(path)
+    try:
+        f = file(path, "wb")
```

Comment:
I see you only copied this, but can we please go for `with open(...)` here as well? Also, I think we don't need to `str()` the content, this would even break in Python 3.